src/video_core/renderer_opengl/gl_stream_buffer.h

```diff
@@ -11,39 +11,51 @@ namespace OpenGL {
 
 class Driver;
 
-class OGLStreamBuffer : private NonCopyable {
+class StreamBuffer {
+    static constexpr std::size_t SYNC_POINTS = 16;
```

Comment:
```suggestion
    static constexpr std::size_t MAX_SYNC_POINTS = 16;
```
maybe makes more sense ?